app/views/users/show.html.erb

```diff
@@ -1 +1,28 @@
-<h1>Here is a list of posts for a given user</h1>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <style>
+
+    </style>
+  </head>
+
+  <body>
+   <h1>Here is the list of Users<h1/>
+    <section>
+        <div>
+        <img  src="<%= @user.photo %>" alt="Avatar" />
+         <h2><%= @user.name %></h2>
+         <h6>
+         <% if @user.postscounter > 0 %>
+         <p>Number of post(s): <%= @user.postscounter %></p>
+            <% end %>
+         </h6>
+        </div>
+    </section>
+
+    <button>
+        <a href='/users/<%= @user.id%>/posts'>See all posts</a>
+    </button>
+  </body>
+</html>
```

Comment:
- [ ] Kindly ensure this User show page matches the wireframe.

  Your Implementation | Wireframe
  :---:|:---:
  ![ifffo](https://user-images.githubusercontent.com/36542986/225569086-4a07d843-0a1b-4fe1-b241-1cdd4aa543f6.jpg) | ![ofoff](https://user-images.githubusercontent.com/36542986/225569097-f1af87cc-ab76-404e-a4ef-e7c2ac9c3e04.jpg)